bazel/repo/clang_tidy_wrapper.sh.tpl

```diff
@@ -2,7 +2,23 @@
 
 set -e
 
+# HACK(chenhao94): Skip all the "textual" sources.
+if printf "%s\n" "$@" | grep "\.txt\.h\(pp\)\?$" > /dev/null; then
```

Comment:
IIRC ? is not POSIX, use 2 -e instead.

Reply:
It is. See the [document](https://en.wikibooks.org/wiki/Regular_Expressions/POSIX-Extended_Regular_Expressions)